src/libraries/Microsoft.NETCore.Platforms/src/runtimeGroups.props

```diff
@@ -232,7 +232,7 @@
     <RuntimeGroup Include="tizen">
       <Parent>linux</Parent>
       <Architectures>x86;armel;arm64</Architectures>
-      <Versions>4.0.0;5.0.0;5.5.0;6.0.0</Versions>
+      <Versions>4.0.0;5.0.0;5.5.0;6.0.0;6.5.0;7.0.0</Versions>
```

Comment:
I'm wondering if we really need all the minor versions. We switched to just tracking major in some other cases.

Reply:
Thanks for your comment!
They are all major Tizen Platform versions, but yes I would see if I can remove unnecessary versions on the future changes.